rotkehlchen/api/v1/parser.py

```diff
@@ -110,10 +110,10 @@ def parse(  # type: ignore  # we have added the resource_object on top of parse
             )
         return data
 
-    def _get_schema(
+    def _get_schema(  # type: ignore
             self,
             argmap: ArgMap,
-            resource_object: Resource,
+            resource_object: MethodView,
```

Comment:
Why did you change the type here to `MethodView`?

Adding type: ignore should only be reserved for bugs and things that can't be solved. The type here seems wrong.

```suggestion
            resource_object: Request,
```

Also remove #type: ignore since by adding the proper type mypy works fine